client-app/src/examples/portfolio/PortfolioPanelModel.ts

```diff
@@ -28,6 +34,21 @@ export class PortfolioPanelModel extends HoistModel {
 
     constructor() {
         super();
+        makeObservable(this);
+        this.initAsync();
+    }
+
+    async initAsync() {
+        return this._initAsync()
+            .catch(e => {
+                XH.handleException(e, {showAlert: false});
+                this.initError = e;
+                throw e;
+            })
+            .linkTo(this.initTask);
+    }
+
+    private async _initAsync() {
```

Comment:
I think our convention here would be to call this `doInitAsync` rather than the underscore (that tends to be used more for private properties than methods).  But also curious why split this out at all.  Seems like it could all just go in `initAsync`